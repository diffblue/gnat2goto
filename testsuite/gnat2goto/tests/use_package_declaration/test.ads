package Test
is
   type Range_10 is range 1..10;
end Test;
