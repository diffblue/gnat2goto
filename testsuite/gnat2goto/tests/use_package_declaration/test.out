[use_package.assertion.1] line 8 Ada Check assertion: SUCCESS
[use_package.assertion.2] line 8 assertion Small_Number * 2 = 8: SUCCESS
VERIFICATION SUCCESSFUL
